src/components/modal/modal.js

```diff
@@ -604,10 +604,13 @@ export const BModal = /*#__PURE__*/ Vue.extend({
     },
     onEnter() {
       this.isBlock = true
+      // We add show class 1 frame after
+      requestAF(() => {
```

Comment:
@tmorehouse Is one `requestAF()` enough here?

Reply:
Yup.

Just enough to trigger the browser CSS show animation/transitions to start - one frame after the content is inserted (or display:none removed)

Reply:
Vue transition `enter-class` is added one frame after insert (or removal of display: none)

Reply:
But we can't use enter-to-class, as Vue immediately removes it after the enter transition completes.

Reply:
Ok 👍 I've tried @philipmountifield's demo and he had to in there to mimic the `nextFrame()` util.